lightningd/onchain_control.c

```diff
@@ -1104,7 +1141,8 @@ static void handle_onchaind_spend_to_us(struct channel *channel,
 	}
 
 	/* No real deadline on this, it's just returning to our wallet. */
-	info->deadline_block = infinite_block_deadline(channel->peer->ld->topology);
+	info->deadline_block =
+	    slow_sweep_deadline(channel->peer->ld->topology, channel);
```

Comment:
Let's just replace all calls to infinite_block_deadline with slow_sweep_deadline.  THe others are mostly "doesn't happen" but still.

Reply:
Will do in a separate PR, after checking we don't rely on anything in those calculations.